README.md

```diff
@@ -295,8 +295,29 @@ print("Probability of Profit (PoP): %.1f%%" % (out["ProbabilityOfProfit"]*100.0)
 
 ## Contributions
 
-Although functional, **OptionLab** is still in its early stages of development. The author has limited time available to work on this library, which is why contributions from individuals with expertise in options and Python programming are greatly appreciated. 
```

Comment:
I think this ("**OptionLab** is still in its early stages of development") is not the case anymore. I mean, the expected functionalities are already available. But we can see that later.